content/en/integrations/guide/aws-terraform-setup.md

```diff
@@ -12,158 +11,185 @@ further_reading:
 
+{{< site-region region="us,us3,us5,eu,gov" >}}
 2. Set up your Terraform configuration file using the example below as a base template. Ensure to update the following parameters before you apply the changes:
    * `AWS_PERMISSIONS_LIST`: The IAM policies needed by Datadog AWS integrations. The current list is available in the [Datadog AWS integration][1] documentation.
    * `AWS_ACCOUNT_ID`: Your AWS account ID.
 
-   See the [Datadog AWS integration resource][2] page in the Terraform registry for further example usage and the full list of optional parameters, as well as additional Datadog resources.
-
-   ```hcl
-   data "aws_iam_policy_document" "datadog_aws_integration_assume_role" {
-      statement {
-      actions = ["sts:AssumeRole"]
-
-      principals {
-         type = "AWS"
-         identifiers = ["arn:aws:iam::464622532012:root"]
-      }
-      condition {
-         test = "StringEquals"
-         variable = "sts:ExternalId"
-
-         values = [
-            "${datadog_integration_aws.sandbox.external_id}"
-         ]
-      }
-      }
-   }
-
-   data "aws_iam_policy_document" "datadog_aws_integration" {
-      statement {
-      actions = [<AWS_PERMISSIONS_LIST>]
-
-      resources = ["*"]
-      }
-   }
-
-   resource "aws_iam_policy" "datadog_aws_integration" {
-      name = "DatadogAWSIntegrationPolicy"
-      policy = "${data.aws_iam_policy_document.datadog_aws_integration.json}"
-   }
-
-   resource "aws_iam_role" "datadog_aws_integration" {
-      name = "DatadogAWSIntegrationRole"
-      description = "Role for Datadog AWS Integration"
-      assume_role_policy = "${data.aws_iam_policy_document.datadog_aws_integration_assume_role.json}"
-   }
-
-   resource "aws_iam_role_policy_attachment" "datadog_aws_integration" {
-      role = "${aws_iam_role.datadog_aws_integration.name}"
-      policy_arn = "${aws_iam_policy.datadog_aws_integration.arn}"
-   }
-
-   resource "aws_iam_role_policy_attachment" "datadog_aws_integration_security_audit" {
-      role = "${aws_iam_role.datadog_aws_integration.name}"
-      policy_arn = "arn:aws:iam::aws:policy/SecurityAudit"
-   }
-
-   resource "datadog_integration_aws" "sandbox" {
-      account_id  = "<AWS_ACCOUNT_ID>"
-      role_name   = "DatadogAWSIntegrationRole"
-   }
-   ```
-
-   [1]: /integrations/amazon_web_services/?tab=manual#aws-iam-permissions
-   [2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/integration_aws
+See the [Terraform Registry][2] for further example usage and the full list of optional parameters, as well as additional Datadog resources.
+
+```hcl
+data "aws_iam_policy_document" "datadog_aws_integration_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::464622532012:root"]
```

Comment:
This would technically be different for gov: https://docs.datadoghq.com/integrations/guide/aws-manual-setup/?tab=roledelegation&site=gov#aws-iam-role-for-datadog